packages/office-ui-fabric-react/src/components/pickers/BasePicker.tsx

```diff
@@ -306,9 +312,13 @@ export class BasePicker<T, P extends IBasePickerProps<T>> extends BaseComponent<
       }
     } else if (suggestionsPromiseLike && suggestionsPromiseLike.then) {
       if (!this.loadingTimer) {
-        this.loadingTimer = this._async.setTimeout(() => this.setState({
-          suggestionsLoading: true
-        }), 500);
+        this.loadingTimer = this._async.setTimeout(() => {
```

Comment:
The better way is to `clearTimeout(this.loadingTimer)` in `unmount`.

Reply:
I don't think it will work. There will always be a host element there, only the expanding card part will be mount and unmount.